templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -84,6 +84,16 @@ contents:
         echo "MTU found for iface: ${iface}: ${iface_mtu}"
       fi
 
+      # store old conn for later
+      old_conn=$(nmcli --fields UUID,DEVICE conn show --active | grep ${iface} | awk '{print $1}')
+
+      extra_brex_args=""
+      # check for dchp-client-id
+      dhcp_client_id=$(nmcli --get-values ipv4.dhcp-client-id conn show ${old_conn})
+      if [ -n "$dhcp_client_id" ]; then
+        extra_brex_args+="ipv4.dhcp-client-id ${dhcp_client_id} "
+      fi
+
```

Comment:
Do we also need to preserve `ipv6.dhcp-duid` ?

Reply:
yeah will add